test/gtest/solver_bwd.hpp

```diff
@@ -175,6 +181,6 @@ struct ConvBwdSolverTest
     miopen::Allocator::ManageDataPtr wei_dev;
     miopen::Allocator::ManageDataPtr out_dev;
     Workspace wspace{};
-    miopenConvFwdAlgorithm_t algo = miopenConvolutionFwdAlgoDirect;
-    bool test_skipped             = false;
+    miopenConvAlgorithm_t algo = miopenConvolutionAlgoDirect;
+    bool test_skipped          = false;
```

Comment:
Is it ever used?
At the first glance you've already removed the case where we skip it in the middle of the execution, so I'd suggest removing `test_skipped` at all.
Also I'd recommend moving skipping logic as close to `SetUp`\`TestSetUp` as possible, because those `SetUp` routines do a lot of memory copy and allocations and tensor initializations which will be thrown away later.

(the same recommendation is for the other solvers)